config/application.rb

```diff
@@ -20,7 +20,6 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
     config.active_job.queue_adapter = :sidekiq
-    config.time_zone = "Kolkata"
```

Comment:
Why removed timezone ?

Reply:
Sometimes what happens, when we store the attendance time like 4/9/2019 then it stores it as 3/9/2019, it may because of the +530 time offset. So when I removed time zone then given time was storing in the database that I have sent in params.

Reply:
> Sometimes what happens, when we store the attendance time like 4/9/2019 then it stores it as 3/9/2019, it may because of the +530 time offset. So when I removed time zone then given time was storing in the database that I have sent in params.

Restored the timezone.